### PR TITLE
DOC: add print statement in to_latex example

### LINKED ISSUE #27789: DOC: add print statement in to_latex example
The docstring of `to_latext` shows a small example (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_latex.html) as 

```
>>> df = pd.DataFrame({'name': ['Raphael', 'Donatello'],
...                    'mask': ['red', 'purple'],
...                    'weapon': ['sai', 'bo staff']})
>>> df.to_latex(index=False) # doctest: +NORMALIZE_WHITESPACE
'\\begin{tabular}{lll}\n\\toprule\n      name &    mask &    weapon
\\\\\n\\midrule\n   Raphael &     red &       sai \\\\\n Donatello &
 purple &  bo staff \\\\\n\\bottomrule\n\\end{tabular}\n'
```

I think it would be nice to also add a print statement like `print(df.to_latex(index=False))`(typically you want to be able to copy it to your tex file, and so you first need to print it to have something you can copy paste)

### PR DESCRIPTION
**Pull request by sameshl**
changed `df.to_latex` to `print(df.to_latex)`

closes #27789

### CODE REVIEW

File: pandas/core/generic.py [hunk 1]
```diff
@@ -2993,10 +2993,17 @@ def to_latex(
         >>> df = pd.DataFrame({'name': ['Raphael', 'Donatello'],
         ...                    'mask': ['red', 'purple'],
         ...                    'weapon': ['sai', 'bo staff']})
-        >>> df.to_latex(index=False) # doctest: +NORMALIZE_WHITESPACE
-        '\\begin{tabular}{lll}\n\\toprule\n      name &    mask &    weapon
-        \\\\\n\\midrule\n   Raphael &     red &       sai \\\\\n Donatello &
-         purple &  bo staff \\\\\n\\bottomrule\n\\end{tabular}\n'
+        >>> print(df.to_latex(index=False)) # doctest: +NORMALIZE_WHITESPACE
+        '\begin{tabular}{lll}
```
Comment (jorisvandenbossche) [inline pandas/core/generic.py]: I think the single quote (the first character) should also be removed (isn't showed by python when printing)

File: pandas/core/generic.py [hunk 2]
```diff
@@ -2993,10 +2993,17 @@ def to_latex(
...
+         \toprule
+               name &    mask &    weapon \\
+         \midrule
+            Raphael &     red &       sai \\
+        Donatello &  purple &  bo staff \\
+        \bottomrule
+        \end{tabular}
+
+        '
```
Comment (jorisvandenbossche) [inline pandas/core/generic.py]: and the same here

**Review by jorisvandenbossche — COMMENTED** [review]
Thanks for the PR! Added a small comment

**Comment by sameshl** [comment]
@jorisvandenbossche Done!

File: pandas/core/generic.py [hunk 3]
```diff
@@ -2993,10 +2993,16 @@ def to_latex(
...
-        '\\begin{tabular}{lll}\n\\toprule\n      name &    mask &    weapon
-        \\\\\n\\midrule\n   Raphael &     red &       sai \\\\\n Donatello &
-         purple &  bo staff \\\\\n\\bottomrule\n\\end{tabular}\n'
+        >>> print(df.to_latex(index=False)) # doctest: +NORMALIZE_WHITESPACE
+        \begin{tabular}{lll}
+         \toprule
+               name &    mask &    weapon \\
+         \midrule
+            Raphael &     red &       sai \\
+        Donatello &  purple &  bo staff \\
```
Comment (jorisvandenbossche) [inline pandas/core/generic.py]: For me the `&` are aligned in the output, like

```
In [14]: print(df.to_latex(index=False))                                                                                                                      
\begin{tabular}{lll}
\toprule
      name &    mask &    weapon \\
\midrule
   Raphael &     red &       sai \\
 Donatello &  purple &  bo staff \\
\bottomrule
\end{tabular}
```

File: pandas/core/generic.py [hunk 4]
```diff
@@ -2993,10 +2993,16 @@ def to_latex(
...
+        \bottomrule
+        \end{tabular}
+
```
Comment (jorisvandenbossche) [inline pandas/core/generic.py]: and a very small detail: can you remove this trailing white line?

Comment (sameshl) [reply pandas/core/generic.py]: I kept it because it looked like it was there in the output when I tried it

Comment (sameshl) [reply pandas/core/generic.py]: But removed now

**Comment by sameshl** [comment]
@jorisvandenbossche I am not able to understand what is causing the test failure. Can you help me out

**Comment by jorisvandenbossche** [comment]
@sameshl that seems as an unrelated failure that you can ignore (I need to check if there is already an issue about this failure, it might be a flaky test)

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
@sameshl Thanks a lot!

**Pull request merged**